### DISCUSSION TITLE
open in visual studio 2019

### DISCUSSION THREAD

**Discussion by dotnethii** [edumeet/edumeet#924] [Ideas]
thanks for program.Please run in visual studio 2019.

**Comment by so010** [comment]
Some of us are using Visual Studio Code if that is an alternative. I don't know if any of us developing under Windows but should be no problem to get other IDEs to work. Setup for nodejs/javascript, configure eslint, ...
Since there are some components which have to be in place to run edumeet locally (docker, TURN server, firewall rules) it could be challenging under Windows. But Windows has WSL now and Docker should work as well so you can give it a try. The compose/README.md under the develop branch https://github.com/edumeet/edumeet/tree/develop/compose should be a good place to start. Would be nice to get some feedback if you get it running. So we can write a how to setup developer enviroment for Windows.
